lib/github.js

```diff
@@ -39,6 +39,8 @@ export default {
   createStatus: async (owner, repo, head_sha, entries) => {
     const github = await getRestClient();
 
+    const links = `### Download link${entries.length > 1 ? "s" : ""}:` + "\n" + entries.map(entry => `- [${entry.artifactId}](${entry.url})`).join("\n")
+    const inputFormat = "### Plugin Installation Manager input format:\n```" + entries.map(entry => entry.hpi).join("") + "```\n([documentation](https://github.com/jenkinsci/plugin-installation-manager-tool/#plugin-input-format))"
```

Comment:
This should be conditional on there being an hpi. Jenkins core and a few others won’t have one

Reply:
Yeah I realized that just before going to sleep, will fix that soon.

Reply:
Addressed in https://github.com/jenkins-infra/incrementals-publisher/pull/95